PulseSampleApp/app/build.gradle

```diff
@@ -0,0 +1,70 @@
+apply plugin: 'com.android.application'
+
+android {
+    buildToolsVersion "21.1.2"
+    compileSdkVersion 23
+
+    defaultConfig {
+        applicationId "com.ooyala.sample.PulseSampleApp"
+        minSdkVersion 16
+        targetSdkVersion 21
+
+        ndk {
+            // React Native for Android is incompatible with 3rd-party 64-bit libraries.
+            abiFilters "armeabi-v7a", "x86"
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+        }
+    }
+}
+
+
+task copyAssetsTask1(type: Copy) {
```

Comment:
Our applications generally have a copy of the libs into the application (https://github.com/ooyala/android-sample-apps/blob/stable/BasicPlaybackSampleApp/app/build.gradle#L22)  That way we don't need to keep extra copies of the jars in the sample applications.  Would you be okay changing to that model?